bundler/lib/bundler/cli/gem.rb

```diff
@@ -79,6 +79,7 @@ def run
       ]
 
       templates.merge!("gitignore.tt" => ".gitignore") if Bundler.git_present?
+      templates.merge!(".rubocop.yml.tt" => ".rubocop.yml")
```

Comment:
This file should only be added if the `--rubocop` flag is passed or enabled in configuration. So, move this line under the relevant condition below and adapt the specs accordingly.

Reply:
:sob: 
Shall do!

Reply:
Thanks! :purple_heart:

Reply:
Should be done! :smile: